src/test/java/seedu/address/model/person/CapTest.java

```diff
@@ -0,0 +1,25 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+class CapTest {
+    // null parameters of Cap() will be tested upon parsing Cap values, as double cannot be null.
+    @Test
+    void isValidCap() {
+        // invalid cap
+        assertFalse(Cap.isValidCap(-5.0, 5.0)); // negative cap value
```

Comment:
Minor, but recommended to add in tests for alphabetical inputs and symbols eg. "asd" and "!@#"